mcp_server/real_mcp_server.py

```
@@ -0,0 +1,91 @@
+                }
+
+            return unknown_tool
+
+    async def process_data(self, data_path: str, operation: str) -> Dict[str, Any]:
+        """Process data with the specified operation"""
+        from protocols.data_processor import task
+
+        return task(data_path=data_path, operation=operation)
+
+    async def execute_code(
+        self, code: str = "", language: str = "python", **kwargs
```

Ruff (0.11.9)

Missing type annotation for `**kwargs`

(ANN003)